src/ui/components/DialogSharedEntryPermissions/components/PermissionButton/PermissionButton.scss

```diff
@@ -0,0 +1,37 @@
+.permission-button {
```

Comment:
It is worth using more specific selectors. For common application components, we try to use the `dl-` prefix.